test/testParams.ts

```diff
@@ -0,0 +1,3 @@
+export const debugDir = '.debug'
+export const infuraIpfsApiUrl = 'https://ipfs.infura.io:5001'
+export const infuraIpfsGateway = 'https://ipfs.infura.io/ipfs/'
```

Comment:
I wonder if we shouldn't spin up a local IPFS node as part of the testing instead... doing this is not great to the public gateway :)

Reply:
I'm 50/50 on this. While we could spin a local node, actually using a remote public node is a very common way this plugin will be used. The content uploaded should be deterministic since the build always produces the same directory so we shouldn't pollute the public node with garbage, just a few KB at most. What do you think?

Reply:
I'd say that selecting a remote IPFS API is really an integration or E2E test we'd want to write and run separately (perhaps against our own exposed service, once available).

Although most of these tests are already integration tests (rather than pure units), it would still be ideal if the main tests didn't need to use the network.

Reply:
We looked to use: [ipfsd-ctl](https://github.com/ipfs/js-ipfsd-ctl) and the library seems broken at the moment. We will stick with infura for now and add an issue to use a local IPFS.